src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RijndaelManaged.cs

```diff
@@ -0,0 +1,110 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+
+namespace System.Security.Cryptography
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public sealed class RijndaelManaged : Rijndael
```

Comment:
How does RijndaelManaged differ from RijndaelImplementation?  Can we get rid of the latter and just have Rijndael.Create instantiate the former?  Or if not, can they be consolidated in some way?